src/helm/benchmark/run_specs.py

```diff
@@ -2243,6 +2289,226 @@ def get_anthropic_hh_rlhf_spec(num_respondents: int, subset: str) -> RunSpec:
     )
 
 
+@run_spec_function("decodingtrust_stereotype_bias")
+def get_stereotype_bias_spec(task: str) -> RunSpec:
+    scenario_spec = ScenarioSpec(
+        class_name="helm.benchmark.scenarios.decodingtrust_stereotype_prompts_scenario"
```

Comment:
I would recommend making the names consistent. Right now you have `decodingtrust_stereotype_bias` and `decodingtrust_stereotype_prompts_scenario`. Rename everything to use either `decodingtrust_stereotype_bias` or ``decodingtrust_stereotype_prompts`.

Reply:
Thanks for the suggestions. This has been fixed in commit 86e39ad885e9dadabaf1b20a4921deb878178e97.